torch/_inductor/runtime/hints.py

```diff
@@ -64,7 +67,7 @@ def AttrsDescriptorWrapper(
             assert res.property_values["tt.equal_to"] == 1
             return res
 
-    except ImportError:
+    elif hasattr(triton.compiler.compiler, "AttrsDescriptor"):
```

Comment:
Correctness - Should this always be defined? (and if not we should throw?). Otherwise if triton is available, but somehow we're missing AttrsDescriptor on triton.compiler.compiler, you'll end up with import errors further down the stack that are less descriptive.

Reply:
Good point, I'll make this `else` instead of `elif`.

And whether `triton.compiler.compiler.AttrsDescriptor` is always defined - it was removed on Jan 1. So my follow-up PR is going to be to add handling for the new semantics.